include/timeline/TimelineView.h

```diff
@@ -59,7 +59,8 @@ struct PendingMessage
           , filename(filename)
           , event_id(event_id)
           , widget(widget)
-        {}
+        {
```

Comment:
I think those were fine. There are some issues with the clang-format running on travis (macos) and the one on liinux, so this will fail on travis. Not sure if this behaviour can be changed with a flag.

Reply:
OK I will revert these lines.